src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public abstract class Machine {
+
```

Comment:
redundant empty line

Reply:
when i delete this empty line and prescribed in the terminal "mvn clean packge"   i got "build failure"